Formula/cbmc-viewer.rb

```diff
@@ -0,0 +1,55 @@
+class CbmcViewer < Formula
+  include Language::Python::Virtualenv
+  desc "Scans the output of CBMC and produces a browsable summary of the results"
+  homepage "https://github.com/awslabs/aws-viewer-for-cbmc"
+  url "https://github.com/awslabs/aws-viewer-for-cbmc.git",
+      tag:      "viewer-2.10",
+      revision: "3049a3451d9c5651c7be1596ddaa69e0051f83c8"
+  license "Apache-2.0"
+
+  depends_on "cbmc" => :test
+  depends_on "python@3.9"
+  depends_on "universal-ctags"
+
+  resource "Jinja2" do
+    url "https://files.pythonhosted.org/packages/91/a5/429efc6246119e1e3fbf562c00187d04e83e54619249eb732bb423efa6c6/Jinja2-3.0.3.tar.gz"
+    sha256 "611bb273cd68f3b993fabdc4064fc858c5b47a973cb5aa7999ec1ba405c87cd7"
+  end
+
+  resource "MarkupSafe" do
+    url "https://files.pythonhosted.org/packages/bf/10/ff66fea6d1788c458663a84d88787bae15d45daa16f6b3ef33322a51fc7e/MarkupSafe-2.0.1.tar.gz"
+    sha256 "594c67807fb16238b30c44bdf74f36c02cdf22d1c8cda91ef8a0ed8dabf5620a"
+  end
+
+  resource "voluptuous" do
+    url "https://files.pythonhosted.org/packages/c0/2c/ccbeb25364e3e0c5e4522f13d66e2fc639bb4d4ecdf73be0959552cbecb4/voluptuous-0.12.2.tar.gz"
+    sha256 "4db1ac5079db9249820d49c891cb4660a6f8cae350491210abce741fabf56513"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    (testpath/"main.c").write <<~EOS
+      #include <stdlib.h>
+
+      static int global;
+
+      int main() {
+        int *ptr = malloc(sizeof(int));
+        assert(global > 0);
+        return 0;
+      }
+    EOS
+    system "goto-cc", "-o", "main.goto", testpath/"main.c"
+    system "cbmc", "main.goto", "--trace", "--xml-ui", ">", "cbmc.xml"
```

Comment:
Seems like it's unable to open `main.c` as this fails with error when i try `brew test cbmc-viewer` - 
(It works fine on terminal)
```
cbmc.xml

<?xml version="1.0" encoding="UTF-8"?>
<cprover>
<program>CBMC 5.51.0 (cbmc-5.51.0)</program>
<message type="STATUS-MESSAGE">
  <text>CBMC version 5.51.0 (cbmc-5.51.0) 64-bit x86_64 linux</text>
</message>

<message type="ERROR">
  <text>Failed to open input file '&gt;'</text>
</message>

</cprover>
```

Reply:
When passing files with abs path, getting `failed to execute` error
```
==> Testing cbmc-viewer
==> goto-cc -o /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/main.goto /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/main.c
==> cbmc /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/main.goto --trace --xml-ui > /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/cbmc.xml
Last 15 lines from /home/ronakf/.cache/Homebrew/Logs/cbmc-viewer/test.02.cbmc.xml:
2022-02-24 18:59:44 +0000

cbmc /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/main.goto --trace --xml-ui > /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/cbmc.xml

Error: cbmc-viewer: failed
An exception occurred within a child process:
  BuildError: Failed executing: cbmc /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/main.goto --trace --xml-ui > /tmp/cbmc-viewer-test-20220224-15911-pmiz7t/cbmc.xml
```